proto/provenance/expiration/v1/tx.proto

```diff
@@ -0,0 +1,72 @@
+// Since: provenance 1.12
+syntax = "proto3";
+package provenance.expiration.v1;
+
+import "gogoproto/gogo.proto";
+import "provenance/expiration/v1/expiration.proto";
+
+option go_package          = "github.com/provenance-io/provenance/x/expiration/types";
+option java_package        = "io.provenance.expiration.v1";
+option java_multiple_files = true;
+
+// Msg defines the Expiration Msg service.
+service Msg {
+  // AddExpiration defines a method for adding an expiration of a module asset
+  rpc AddExpiration(MsgAddExpirationRequest) returns (MsgAddExpirationResponse);
+  // ExtendExpiration defines a method for extending an expiration of a module asset
+  rpc ExtendExpiration(MsgExtendExpirationRequest) returns (MsgExtendExpirationResponse);
+  // InvokeExpiration defines a method for invoking expiration logic for a module asset
+  rpc InvokeExpiration(MsgInvokeExpirationRequest) returns (MsgInvokeExpirationResponse);
+}
+
+// MsgAddExpirationRequest represents a message to add an expiration
+message MsgAddExpirationRequest {
+  option (gogoproto.equal)            = false;
+  option (gogoproto.goproto_stringer) = false;
+  option (gogoproto.stringer)         = true;
+  option (gogoproto.goproto_getters)  = false;
+
+  // expiration is the Expiration you want to add
+  Expiration expiration = 1 [(gogoproto.nullable) = false];
+  // signers is the list of addresses of those signing this request.
+  repeated string signers = 2;
```

Comment:
An expiration can only have one owner. So, there's no need to have multiple signers, is there?

Reply:
Yes, the expiration module supports `grants`. One reason for adding `authz` support is the metadata module. The other is that it would be nice to support deleting of assets with granted permissions.